MHVLib/Models/Generated/Categories/MHVGoalsApi.m

```diff
@@ -326,45 +288,37 @@ + (NSURLSessionTask*)patchGoalsWithGoalsWrapper:(MHVGoalsWrapper*)goalsWrapper
             NSError* error = [NSError errorWithDomain:kMHVGoalsApiErrorDomain code:kMHVGoalsApiMissingParamErrorCode userInfo:userInfo];
             completion(nil, error);
         }
-        return nil;
     }
-
+    
     NSMutableString* resourcePath = [NSMutableString stringWithFormat:@"/Goals"];
-
+    
     NSMutableDictionary *pathParams = [[NSMutableDictionary alloc] init];
-
+    
     NSMutableDictionary* queryParams = [[NSMutableDictionary alloc] init];
-
+    
     id bodyParam = nil;
     NSMutableDictionary *formParams = [[NSMutableDictionary alloc] init];
     bodyParam = goalsWrapper;
-
-    return [MHVRemoteMonitoringClient requestWithPath: resourcePath
-                                    method:@"PATCH"
-                                pathParams:pathParams
-                               queryParams:queryParams
-                                formParams:formParams
-                                      body:bodyParam
-                                   toClass:[MHVGoalsResponse class]
-                                completion:^(id data, NSError *error)
-                                {
-                                    if(completion)
-                                    {
-                                        completion((MHVGoalsResponse*)data, error);
-                                    }
-                                }
-            ];
+    
+    [self requestWithPath:resourcePath
+               httpMethod:@"PATCH"
+               pathParams:pathParams
+              queryParams:queryParams
+               formParams:formParams
```

Comment:
Nothing uses the formParams now.  If you don't think it's going to be used, can delete